### PR TITLE
support installing an unencrypted core boot classic system

### PR DESCRIPTION
**Pull request by mwhudson**
I have used code like this (but not exactly this) to install a core boot classic system in a VM.

### CODE REVIEW

File: subiquity/server/controllers/install.py [hunk 1]
```diff
@@ -245,30 +245,49 @@ def make_curtin_step(name, stages, acquire_config, *,
...
+                make_curtin_step(
+                    name="extract", stages=["extract"],
+                    acquire_config=self.acquire_generic_config,
+                    ).run,
+                fs_controller.finish_install,
+                self.setup_target,
+                ])
+        else:
+            steps.extend([
+                make_curtin_step(
```
Comment (dbungert) [inline subiquity/server/controllers/install.py:273]: duplication with if block

**Review by dbungert — APPROVED** [review]
This all looks fine but some sort of automated testing would be nice.

File: subiquity/server/controllers/install.py (hunk 1 above)
Comment (mwhudson) [reply subiquity/server/controllers/install.py:273]: ah yes, that will go away in the next branch

**Comment by mwhudson** [comment]
> This all looks fine but some sort of automated testing would be nice.

Added something and fixed the bug this inevitably revealed...

**Review by dbungert — APPROVED** [review]

**Pull request merged**